vscode-extension/esbuild.script.mjs

```diff
@@ -0,0 +1,26 @@
+import * as esbuild from 'esbuild';
+
+await esbuild.build({
+  entryPoints: ['./src/client.ts'],
+  bundle: true,
+  outfile: 'client.js',
+  platform: 'node',
+  minify: true,
+  target: 'es2017',
```

Comment:
Could this be bumped ahead at all, or are there older v8 versions running in vscodes out there?